apps/expo/src/components/TextInput.tsx

```diff
@@ -0,0 +1,146 @@
+import React, { useState, useMemo } from "react";
+import { View, TextInput, Button, Text, StyleSheet } from "react-native";
+import { TouchableOpacity } from "react-native";
+
+const CustomCheckBox = ({ value, onValueChange }) => (
```

Comment:
Can you import FC from typescript such as `import type { FC } from 'react'` then do the following

```ts
const CustomCheckBox: FC<CustomCheckBoxProps> = ...
```

Also provide the following interface for that object